lib/steamaccount.js

```diff
@@ -78,8 +84,8 @@ module.exports = class SteamAccount extends EventEmitter {
   boost() {
     return this.login()
       .then(() => {
-        this.client.setPersona(SteamUser.EPersonaState.Offline)
-        this.client.gamesPlayed(this.games !== null ? this.games : [10, 730])
+        this.client.setPersona(this.customtitle ? SteamUser.EPersonaState.Online : SteamUser.EPersonaState.Offline)
+        this.client.gamesPlayed(this.games !== null ? this.games : [10, 730])          
```

Comment:
Please remove the trailing spaces here. If you add or change things please run `npm run pretty` afterwards to get the source code auto formatted. ;)